app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -97,6 +99,8 @@ open class HomeActivity : AppCompatActivity() {
 
         setupToolbarAndNavigation()
 
+        finishOnboarding = intent.getBooleanExtra("finishonboarding", false)
```

Comment:
nit: the key should be a constant: see the HomeActivity companion object for examples.

Reply:
Good call!